src/commons/withScrollEnabler.tsx

```diff
@@ -42,7 +42,7 @@ function withScrollEnabler<PROPS extends SupportedViews>(
           }
         }
       },
-      [props.horizontal]
+      [props.horizontal, checkScroll]
```

Comment:
I don't think it's recommended to pass to the dependency array a callback. 
and actually I don't think you even need to, I think `[props.horizontal]` is enough in this case. 

same goes for the onLayout callback

Reply:
That was recommended by the new plugin.